Update release notes for 1.7.0 (#2445)
Automattic/studio: RELEASE-NOTES.txt

Unreleased
==========

1.7.0
=====

> * Added support for managing sites to Studio CLI tool

Reviewer: ```suggestion
* Added support for managing sites in the Studio CLI tool
```

Author: Thanks, fixed.

> * Added WP CLI support to Studio CLI tool

Reviewer 2: ```suggestion
* Added support for managing sites in the Studio CLI tool
* Added WP CLI support to Studio CLI tool
* Added support for logging in to WordPress.com using the CLI tool
* Started bundling a regular Node.js runtime for better and more consistent performance #2303
```
- `studio auth` probably isn't that interesting to end users, but we might as well call it out.
- I believe performance is highly interesting, so it might be worth considering calling out https://github.com/Automattic/studio/pull/2303 specifically

Author: Thanks, added in 15214c120538ad84f4444d725566499ccf72dd82

* Improved Xdebug support (Beta Feature) #2316 #2395, #2410
* Added file upload progress and offline status indicator in Sync UI #2332
* Increased Sync push size limit to 5 GB #2377
* Added support for Blueprint translations #2363

> * Improved speed of creating WordPress site in online mode #2364

Reviewer: ```suggestion
* Improved the speed of creating a WordPress site in online mode #2364
```

Author: Thanks, fixed.

> * Improved Blueprint deeplink step design #2397

Reviewer: ```suggestion
* Improved the design of the Blueprint deeplink step #2397
```

Author: Thanks, fixed.
